macros/schema_tests/expression_is_true.sql

```diff
@@ -1,4 +1,8 @@
 {% macro test_expression_is_true(model, condition='true') %}
```

Comment:
another quirk: in #310, I propose changing the default value for condition to be `1=1` instead of `'true'`. How might I set that default value to be different _only_ for tsql? isn't that something we can't do? In a normal scenario you could edit the `schema.yml`, but in this case tsql can't touch it as it's part of another repo.

Reply:
I wonder if you could do something like:
```
{% macro sqlserver__test_expression_is_true(model, condition) %}

  {# override generic default #}
  {% if condition = 'true' %}
      {% set condition = '1=1' %}
  {% endif %}
  
  {{ return(default__test_expression_is_true(model, condition) }}

{% endmacro %}
```

(although I'm not opposed to the proposed change in #310, making this `1=1` across the board)

Reply:
good idea. i'm going to break out more of these questions into a different PR, so that this PR's focus can just be about dispatching